script.js

```diff
@@ -143,7 +147,7 @@ seeproject.forEach((p) => p.addEventListener('click', (p) => {
       <li class="dev"><img src="./images/Counter.png" alt="dot">&nbsp; ${pop.year}</li>
     </ul>
     </div>
-    <img src="${pop.image}" alt="live">
+    <img src="${pop.image}" alt="live" class="beryl">
```

Comment:
Nice work on the popup 💯 but the spacing isn't quite right, make sure that the margin to the left of the image is the same to its right in order to center the image. Take a look below at the differences 👇🏽 

## Figma

![Screenshot 2023-04-11 at 12 03 11](https://user-images.githubusercontent.com/45739963/231222028-f92f8c0a-078c-4e61-b212-9373a59e7d34.png)

## Your design

![Screenshot 2023-04-11 at 12 06 20](https://user-images.githubusercontent.com/45739963/231222845-9af16ada-baf2-43e5-9b90-140d1dd1d122.png)

Also, make sure to add the `hr` between the list of technologies and the two `buttons` 🤓

Lastly, this is *OPTIONAL* but consider using the `pointer` cursor so that it looks like the buttons are clickable. 😉